app/src/features/datasets/components/RenderWithHandlebars.tsx

```diff
@@ -0,0 +1,21 @@
+import * as DOMPurify from 'dompurify';
+import 'styles/handlebar.scss';
+import { useHandlebars } from 'utils/handlebarsUtils';
+
+export interface IRenderWithHandleBarProps {
+  dataset: any;
+  rawTemplate: any;
```

Comment:
Is there a more specific type we can use for this? Something exported by handlebars maybe?